app/middlewares/user.js

```diff
@@ -9,8 +9,8 @@ exports.userVerifyUserByEmail = async (req, _, next) => {
     const isExistEmail = await userService.findUserByEmail(email);
     if (isExistEmail) return next(errors.badRequestError('Email already exist!'));
     return next();
-  } catch {
-    return next(errors.databaseError('Error to verify email'));
+  } catch (error) {
```

Comment:
I like the last versión of this piece of code hahaha, I would do something like:

```
catch (error) {
  logger.info(error);
  return next(errors.databaseError('Error to verify email'));
}

Reply:
logger applied